NuRadioReco/modules/channelCWNotchFilter.py

```diff
@@ -0,0 +1,227 @@
+import numpy as np
+from scipy import signal
+from NuRadioReco.utilities import units
+from NuRadioReco.utilities import fft
+
+"""
+Contains function to filter continuous wave out of the signal
+functions should work on a per event basis to comply with the iteration methods used in readRNOGData
+"""
+
+
+def find_frequency_peaks_from_trace(trace : np.ndarray, fs : float = 3.2e9 * units.Hz, threshold = 4):
+    """
+    Function fo find the frequency peaks in the real fourier transform of the input trace,
+
+    Parameters
+    ----------
+    trace : np.ndarray
+        waveform (shape: [24,2048])
+    fs: float, default = 3.2e9 Hz
+        sampling frequency of the RNO-G DAQ, (input should be taking from the channel object)
+    threshold : int, default = 4
+        threshold for peak definition. A peak is defined as a point in the frequency spectrum
+        that exceeds threshold * rms(real fourier transform)
+    
+    Returns
+    -------
+    freq_peaks : np.ndarray
+        frequencies at which a peak was found
+    """
+    freq = np.fft.rfftfreq(2048, d = 1/fs)
```

Comment:
Pep8 recommends to not add spaces around the `=` for keyword arguments: https://peps.python.org/pep-0008/#other-recommendations

Reply:
This is everywhere in this file. I let you decide if you want to change it